app/app/urls.py

```diff
@@ -133,6 +134,7 @@
     url(r'^sync/get_amount?', dashboard.helpers.amount, name='helpers_amount'),
     url(r'^sync/get_issue_details?', dashboard.helpers.issue_details, name='helpers_issue_details'),
     url(r'^sync/search_save?', dashboard.views.save_search, name='save_search'),
+    url(r'^sync/get_quickstart_video?', dashboard.views.get_quickstart_video, name='get_quickstart_video'),
```

Comment:
we should maybe come up with another path for modals, since `sync/` only makes sense for frontend/backend syncronoization routes

Reply:
that makes sense to me. @mbeacom ?

Reply:
Yeah, that makes sense to me as well!  Ideally, I'd like to start moving some of the url entries to their respective django app `urls.py` and pull them into `app/urls.py` via `include()`

Reply:
@mbeacom :+1:  for moving to individual django apps and using `include()`